app/Http/Controllers/WikiController.php

```diff
@@ -32,7 +32,7 @@ public function __construct( DomainValidator $domainValidator )
     public function create(Request $request): \Illuminate\Http\Response
     {
         $user = $request->user();
-        $submittedDomain = $request->input('domain');
+        $submittedDomain = strtolower($request->input('domain'));
```

Comment:
Not sure about strings in PHP, does this need to be `strtolower` or `mb_strtolower`?

Reply:
either would do, I think there is no point in using mb_strlower which I assume is internally more expensive and complex.

This would be necessary if we were to go on to support multibyte chars in this place but since we filter any of those out with the validator I think it doesn't matter (and I don't think we'd ever want to support these in our domain names).